client/my-sites/stats/promo-cards/index.jsx

```diff
@@ -72,7 +75,10 @@ export default function PromoCards( { isOdysseyStats, slug, pageSlug } ) {
 		// Events need to incorporate the page and the click type.
 		// This will allow us to account for integration across different pages.
 		// FORMAT: calypso_stats_PAGE_mobile_cta_EVENT
-		const tracksEventName = `calypso_stats_traffic_mobile_cta_${ event.replaceAll( '-', '_' ) }`;
+		const tracksEventName = `calypso_stats_${ pageSlug }_mobile_cta_${ event }`.replaceAll(
+			'-',
+			'_'
+		);
```

Comment:
Could we add a code comment here listing all the different permutations of what `tracksEventName` can be?

> "Don’t programmatically create event names. They should be hardcoded so they are easily searchable in the code base."
- From "General Naming Conventions" on PCYsg-4S2-p2.

Reply:
I tried to set up the event names based on those guidelines. Part of the initial design was to have unique events for each page the CTA is shown on which is why we build the event based on the provided pageSlug prop.

There are 6 possible click events per page. I've added a comment listing those out (currently 18 possible events) but it will fall out of date if the PromoCards component is used on any other pages.

I'm not sure I have a better solution than programmatically generating the event names. If you have any suggestions, please let me know.